src/hooks/custom-hooks/__tests__/useExtendedOrderDetails.spec.ts

```diff
@@ -0,0 +1,1064 @@
+import moment from 'moment';
+import { THooks, TServerTime } from 'types';
+import { renderHook } from '@testing-library/react';
+import useExtendedOrderDetails from '../useExtendedOrderDetails';
+
+const mockData: {
+    loginId: string;
+    orderDetails: THooks.Order.Get;
+    serverTime: TServerTime;
+} = {
+    loginId: '123',
+    orderDetails: {
+        account_currency: 'USD',
+        // @ts-expect-error - Only relevant fields are included
+        advertiser_details: {
+            loginid: '',
+        },
+        amount_display: '100',
+        type: 'buy',
+    },
+    serverTime: undefined,
+};
+
+describe('useExtendedOrderDetails', () => {
```

Comment:
can you please check if we are using all these functions in useExtendedOrderDetails?